### ISSUE TITLE
[docs] mac install (Trac #1914)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1906]
For http://software.icecube.wisc.edu/documentation/projects/cmake/supported_platforms/osx.html#step-by-step-instructions

the package pyqt does not exist anymore.

Migrated from https://code.icecube.wisc.edu/ticket/1914
```json
{
    "status": "closed", 
    "changetime": "2019-01-12T00:33:41", 
    "description": "For http://software.icecube.wisc.edu/documentation/projects/cmake/supported_platforms/osx.html#step-by-step-instructions\n\nthe package pyqt does not exist anymore.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547253221905647", 
    "component": "combo core", 
    "summary": "[docs] mac install", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2016-11-29T17:46:57", 
    "milestone": "Long-Term Future", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161129T17:49:39: david.schultz commented:

> Also, the homebrew.pth file might not be writable if you've never pip installed something before (the directory doesn't exist yet).  The order of those lines should be changed.

**Comment by icecube-trac** [comment]
Trac update at 20161129T18:02:18: david.schultz commented:

> We apparently also need to explicitly list `freetype` as an install package for brew.

**Comment by icecube-trac** [comment]
Trac update at 20161129T19:51:20: david.schultz commented:

> step-by-step needs to mention `boost-python`

**Comment by icecube-trac** [comment]
Trac update at 20161129T19:57:40: 
* david.schultz changed owner from "" to "nega"
* david.schultz changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20171003T13:10:51: 
* nega changed status from "assigned" to "closed"
* nega commented:

> In changeset 2981:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2981"
> fix typo
> 
> closes #1906
> ```


* nega changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:33:41: nwhitehorn commented:

> In changeset 170408:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170408"
> fix typo
> 
> closes #1906
> ```
